### ISSUE TITLE
Offer site-specific conf option to disable typeahead completion in Publish wizard

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3769]
@billreed63 @peller 

I don't think we should offer typeahead for email addresses on maqetta.org. Makes sense for internal deployments, but not for public deployments. Seems like we need a new option in maqetta.conf, and somehow get davinci/review/widgets/PublishWizard.js to honor that option.

**Comment by billreed63** [comment]
I don't think we need another option, we could just key off the authorization type, if we are using LDAP we could search LDAP for type ahead, if not using LDAP we just search the users maqetta address book #3753

**Comment by JonFerraiolo** [comment]
But the problem is that on maqetta.org, users can find the emails of people who might not want to have their emails discovered.

**Comment by peller** [comment]
I think @billreed63 is suggesting that on maqetta.org we would only search the user's defined groups, once that feature is implemented.

**Comment by billreed63** [comment]
Yes, but @JonFerraiolo is correct the current implementation is to search all the users of maqetta

**Comment by JonFerraiolo** [comment]
> I think @billreed63 is suggesting that on maqetta.org we would only search the user's defined groups, once that feature is implemented.

Yes, that's what we would want. Probably a configuration option for typeahead that has values off, all users, or named groups only.
